test/e2e_node/device_plugin.go

```diff
@@ -163,6 +164,109 @@ var _ = framework.KubeDescribe("Device Plugin [Feature:DevicePlugin] [Serial] [D
 	})
 })
 
+// Serial because the test restarts Kubelet
+// Slow because this is a soak test
+var _ = framework.KubeDescribe("Device Plugin Soak [Feature:DevicePluginSoak] [Soak] [Slow] [Serial] [Disruptive]", func() {
+	f := framework.NewDefaultFramework("device-plugin-errors")
+
+	Context("DevicePlugin", func() {
+		By("Enabling support for Device Plugin")
+		tempSetCurrentKubeletConfig(f, func(initialConfig *kubeletconfig.KubeletConfiguration) {
+			initialConfig.FeatureGates[string(features.DevicePlugins)] = true
+		})
+
+		const (
+			// how many waves of device plugins
+			scale = 10
```

Comment:
Rather than run 10 x in the same test, it might be a better model to run the test once here, and use 
[`--start-services` and `--stop-services`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/util.go#L57-L58) to reuse the same kubelet and apiserver in the next run.  That way, we can control the number of times it runs from the test suite, rather than hard-coding it in the test.  We could also find longer-term leaks by running it repeatedly (e.g. weeks).

I think you should be able to re-use the same kubelet and apiserver by specifying `make test-e2e-node ... TEST_ARGS="--stop-services=false"` on the first run, and `make test-e2e-node ... TEST_ARGS="--start-services=false"` on the second run, so that we keep the same "services" for both runs.

Can you try that out and see if it works?  If not, I think this model is acceptable for now, but we will want to move to a more customization model in the future.